src/pages/contributors/sacha-labourey.adoc

```diff
@@ -0,0 +1,185 @@
+
+== What is your background prior to contributing to Jenkins?
+
+I studied engineering in Switzerland.
+During this time, I started my consulting company during my studies to pay for my studies.
+After finishing my studies, I was in a transition phase where I tried to figure out what I wanted to do and where I wanted to go next.
+I knew I wanted to be an entrepreneur but I didn't quite understand what it involved.
+I was missing a user guide on how to get started, so that was not necessarily the happiest time for me.
+
+Eventually, I joined the JBoss open-source team back in 2000/2001.
+JBoss had originally started as a company called Telkel that went bust.
+When this happened, the founder of JBoss, Marc Fleury, was in a similar transition situation.
+He had recently moved with his wife and baby and was starting to give JBoss training to make some money.
+He was traveling to multiple cities because while the project had traction, there was no way to make enough money.
+I wasn't exactly flush with money, but I ended up spending the last few francs that I had in my pocket to attend that training.
+
+This ended up being a sort of "love at first sight" between Marc and I, and that's when I got a lot more involved as a contributor.
+This was a great starting point, and I implemented the first version of clustering for JBoss.
+After getting more acclimated to JBoss, I helped structure a European region.
+Eventually, I joined the board and became CTO, overseeing all of product engineering.
+The company was acquired by Red Hat in 2006, where I stayed for about three to four years as a co-GM for the middleware division.
```

Comment:
What is a co-GM? 🤔

Reply:
I believe it's a co-general manager.

Reply:
Thanks!

Reply:
It may be a good idea to spell out the word as someone else might have the same question.

Reply:
I definitely agree. I actually updated it in the latest version but it didn't seem to update this comment. I opted for just saying "general manager" instead due to the possible confusion. Thanks very much!